_posts/2017-11-14-django-girls-bangalore-2017.md

```diff
@@ -0,0 +1,59 @@
+is small event report from my side.
+
+Django girls is not for profit initiative led by Ola Sitarska and Ola Sendecka.
+Such movement helps women to learn the skills of website development using the
+well-known web-framework Django. This community is backed by organizers from
+many countries of this world. Organizations like The Python Software Foundation,
+Github, DjangoProject and much more are funding Django girls.
+
+Django girls Bangalore chapter was organized by Sourav Singh and Kumar Anirudha.
+It was my second time mentoring for the Django girls event. First was for
+Ahmedabad chapter. The venue was sponsored by HackerEarth. X male and 2 women
+mentored X women during this event. Each mentor was assigned more or less 3
+participants. Introducing newbie with web development becomes easy with the help
+of Django girls handbook. The Django girls handbook is a combination of
+beginner-friendly hands-on tutorials described in a simple language. The
+handbook contains tutorials on basics of Python programming language to
+deploying your application. Pupils under me were already ready by
+pre-configuring Python with their workstation. We started by introducing our
+selves. We took some time browsing the website of undersea cables. One of the
+amusing questions I got is, “Isn't the world connected with Satellites?”. My
+team was comfortable with the Python, so we quickly skimmed to the part where I
+introduced them to the basics of web and then Django. I noticed mentors were
+progressing according to the convenience of participants. Nice amount of time
+was invested in discussing raised queries. During illumination, we heard a loud
+call for the lunch. A decent meal was served to all the members. I networked
+with other mentors and participants during the break. Post-lunch we created a
+blog app and configured it with our existing project. Introducing newbie with
+Django models and ORM took little efforts. With the time as a constraint, I
+focused on admin panel and taught girls about deploying their websites to
+PythonAnywhere. I am happy with the hard work done by my team. They were able to
+demonstrate what they did to the world. I was more joyful than them for such
+achievement.
+
+The closing ceremony turned amusing event for us. 10 copies of Two scopes of
+Django book were distributed to the participants chosen by random draw strategy.
+I solemnly thank the authors of the book and Pothi.com for gifting such a nice
+reference. Participants shared their experiences of the day. Mentors pinpointed
+helpful resources to look after. They insisted girls to not satisfy at such
+point and open their wings by developing websites using skills they learned.
+T-shirts, stickers and badges were distributed as event swags.
+
+You can find the list of all Django girls chapters here. Djangonuts are shouted
+to become a mentor for Django girls events in your town. If you are not finding
```

Comment:
`Djangonuts (Djangonauts?) are calling for new mentors to join Django Girls events in your town.`